tests/test_build/navbar_icon_links.html

```diff
@@ -29,8 +29,13 @@
  <li class="nav-item">
   <a class="nav-link" href="https://site4.org" rel="noopener" target="_blank" title="WRONG TYPE">
    <span>
-    Incorrectly configured icon link. Type must by `fontawesome` or `local`.
+    Incorrectly configured icon link. Type must by `fontawesome`, `url` or `local`.
```

Comment:
By -> be

I know it's not your change but would you mind fixing this typo?

Reply:
Don't mind at all, fixed in https://github.com/pydata/pydata-sphinx-theme/pull/616/commits/0ab7d6da12d1229f3ed9bdfc834f074f447bb2ab